### PR TITLE
Create a page where you can see your meetings and add the option to cancel a meeting

### LINKED ISSUE #133: Create a page where you can see your meetings and add the option to cancel a meeting

### PR DESCRIPTION
**Pull request by idosinger98**
The changes I made are:
1. I added a page for the professional so he can see his future appointments.

2. I added a page for the client so he can see his future appointments.

3. I added the option to cancel an appointment (both the client can cancel an appointment and the professional can cancel an appointment).

Resolve #133 
image

image

### CODE REVIEW

**Review by idosinger98 — COMMENTED** [review]
@Yarboa Can you help me understand why it doesn't recognize pytz, I added it to the pipfile

File: Pipfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 django = "*"
 pillow = "*"
+pytz = "*"
```
Comment (Yarboa) [inline Pipfile:9]:
```suggestion
```

File: Pipfile [hunk 2]
```diff
@@ -6,6 +6,7 @@ name = "pypi"
...
 
 [dev-packages]
 flake8 = "*"
```
Comment (Yarboa) [inline Pipfile:12]:
```suggestion
flake8 = "*"
pytz
```

File: Pipfile [hunk 3]
```diff
@@ -9,6 +9,7 @@ pillow = "*"
...
+pytz
```
Comment (Yarboa) [inline Pipfile:12]:
```suggestion
pytz = "*"
```

**Review by idoyek — APPROVED** [review]
LGTM

**Review by patrisiakaplun — APPROVED** [review]
LGTM

**Review by OfirBachar — APPROVED** [review]
LGTM

File: reservation/test_appointment_model.py [hunk 4]
```diff
@@ -68,10 +68,12 @@ def test_del_appointment(self, make_appointment):
...
-    def test_get_client_list_on_certain_day_by_professional_and_date(self, persisted_appointment_pool, professional):
+    def test_get_appointment_list_after_current_day_by_professional(self, persisted_appointment_pool, professional):
         assert persisted_appointment_pool == \
-               list(Appointment.get_client_list_on_certain_day(professional_id=professional,
-                                                               date=(current_datetime + timedelta(days=5)).date()))
+               list(Appointment.get_appointments_list_after_current_day(profile_id=professional.professional_id,
+                                                                        type_of_user=True))
+
+    def test_get_appointment_list_after_current_day_by_client(self, persisted_appointment_pool, demo_client):
+        assert persisted_appointment_pool[1] == \
```
Comment (Yarboa) [inline reservation/test_appointment_model.py:77]: Please consult with @guybeckenstein how to use parameterized

Comment (guybeckenstein) [reply reservation/test_appointment_model.py:77]: [see lines 38-42 in `landing/tests/test_base_navigation_bar.py`](https://github.com/redhat-beyond/Pro4U/pull/125/files#diff-548f1fdd5149d72034d5ab17fdfa3ebeaf716a4ee544f61c2333d0236dd1a945)

**Comment by guybeckenstein** [comment]
Please remove `demo_client` and use `make_client` instead

**Comment by idosinger98** [comment]
> Please do not use demo_client use make_client make_professional fixtures

Done.

**Comment by idosinger98** [comment]
> Please remove `demo_client` and use `make_client` instead

Done.

File: reservation/test_appointment_model.py (hunk 4 above)
Comment (idosinger98) [reply reservation/test_appointment_model.py]: Done @Yarboa

File: reservation/test_appointment_model.py [hunk 5]
```diff
@@ -29,7 +29,7 @@ def persisted_appointment_pool(make_appointment, demo_client):
                                     summary="")
     save_appointment(appointment)
```
[the remark this answers is not in the document]
Comment (idosinger98) [reply reservation/test_appointment_model.py]: I fixed it @Yarboa

**Review by guybeckenstein — APPROVED** [review]
LGTM

**Review by TalReinfeld — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**